src/components/Balance.vue

```diff
@@ -30,15 +67,32 @@ export default {
 .balance {
   @extend %face-mono-base;
   font-weight: normal;
-  color: $color-neutral-negative-1;
+  color: $color-neutral-negative-3;
+
+  .left {
+    color: $color-neutral-negative-1;
+    font-size: rem(13px);
+  }
+
+  &:not(.convertable) {
+    @extend %face-mono-xs;
+    letter-spacing: normal;
+    font-weight: 500;
+    color: $color-neutral-negative-1;
+  }
 
   &.invert {
     color: #fff;
+
+    .left {
+      color: rgba(255, 255, 255, 0.7);
+    }
   }
 
-  &:after {
-    @extend %face-mono-xs;
-    content: 'AE';
+  @media (max-width: 410px) {
+    .left {
+      display: none;
+    }
```

Comment:
can we hide convert button in this case as well? or maybe find a better icon for this case?